jwst/extract_1d/extract.py

```diff
@@ -2859,14 +2903,29 @@ def do_extract1d(input_model, ref_dict, smoothing_length=None,
             spec.spectral_order = sp_order
             spec.dispersion_direction = extract_params['dispaxis']
             copy_keyword_info(slit, slit.name, spec)
+
+            if source_type is not None and source_type.upper() == 'POINT' and apcorr_table is not None:
+                log.info('Applying Aperture correction.')
+                if instrument == 'NIRSPEC':
```

Comment:
Maybe some comments would be good here to explain why these wl values are being set the way they are.